bindings/java/src/org/sleuthkit/datamodel/AbstractFile.java

```diff
@@ -1315,6 +1328,44 @@ public void save() throws TskCoreException {
 		}
 	}
 
+	/**
+	 * Get the owner uid.
+	 * 
+	 * Note this is a string uid, typically a Windows SID. 
+	 * This is different from the numeric uid commonly found 
+	 * on Unix based file systems.
```

Comment:
It looks like Linux file systems have a concept of a fsuid that is similar to a SID. We may want to comment on this.